janus_core/md.py

```diff
@@ -345,7 +345,7 @@ def get_log_header() -> str:
         """
         log_header = (
             "Step | real time[s] | Time [fs] | Epot/N [eV] | Ekin/N [eV] | "
-            "c_T [eV/K] | Etot/N [eV] | Density [g/cm^3] | Volume [A^3] | "
+            "T [K] | Etot/N [eV] | Density [g/cm^3] | Volume [A^3] | "
```

Comment:
I think the corresponding `c_T` in `get_log_stats` needs updating as well? (Although surely it's not always valid to use 3/2 kT?)

Reply:
do you mean the name of the variable?, formula is fine for 3d systems...

Reply:
> do you mean the name of the variable?, formula is fine for 3d systems...

Yes, it wasn't a particularly good variable name anyway (it required disabling the pylint warning, but I thought it was ok since it matched the header).

(Initially the `c_` and 3/2 factor made me think of heat capacities/internal energy etc. with the different plateaus, but yep it's fine)